src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -2372,11 +2376,8 @@ private boolean isFeedbackSessionViewableTo(
         }
 
         // Allow all instructors to view always
-        InstructorAttributes instructor = instructorsLogic.
-                getInstructorForEmail(session.courseId, userEmail);
-        if (instructor != null) {
-            return instructorsLogic.isEmailOfInstructorOfCourse(
-                    instructor.email, session.courseId);
+        if (instructorOfCourse != null) {
+            return true;
         }
```

Comment:
One remedy is to pass a `boolean isInstructor` parameter instead of the `instructorOfCourse` object (because the only use of the `instructorOfCourse` object within this method is to check if it is null.